app/controllers/admin/community_customizations_controller.rb

```diff
@@ -5,11 +5,16 @@ def edit_details
     @selected_left_navi_link = "tribe_details"
     # @community_customization is fetched in application_controller
     @community_customizations ||= find_or_initialize_customizations(@current_community.locales)
+    all_locales = MarketplaceService::API::Marketplaces.all_locales
+    enabled_locale_keys = available_locales().map{ |locale| locale[1] }
```

Comment:
Ruby trick:

There's a method `second`, which can be used to pick the second item from the array

```ruby
available_locales().map{ |locale| locale.second }
```

and then there's a shorthand for that:

```ruby
available_locales().map(&:second)
```